src/Mapbender/CoreBundle/Asset/TranslationCompiler.php

```diff
@@ -48,4 +61,44 @@ protected function getTemplate()
     {
         return '@MapbenderCoreBundle/Resources/public/mapbender.trans.js';
     }
+
+    /**
+     * @param string $template
+     * @return string[]
+     */
+    protected function extractFromTemplate($template)
+    {
+        return json_decode($this->templateEngine->render($template), true);
+    }
+
+    /**
+     * @param string $input translation key or prefix pattern ending in '.*'
+     * @return string[]
+     */
+    protected function translatePattern($input)
+    {
+        $values = array();
+        $allMessages = null;
+        if (preg_match('/\*$/', $input)) {
+            $wildcardPrefix = rtrim($input, '*');
+            if (!$wildcardPrefix || false !== strpos($wildcardPrefix, '*')) {
+                throw new \RuntimeException("Invalid translation key input " . print_r($input, true));
+            }
+            if (!$allMessages) {
```

Comment:
Am I blind or is this check unnecessary? When does $allMessage get any other value then null?

Reply:
You're right. This was supposed to be a one-time preinitialization but it didn't work that way.
Fixed in [b4ea9f4](https://github.com/mapbender/mapbender/commit/b4ea9f417612cbb5eb4a7dfa26515e89c15109b4).